pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -151,7 +151,12 @@ func (moa *MultiOrgAlertmanager) SyncAlertmanagersForOrgs(ctx context.Context, o
 	for _, orgID := range orgIDs {
 		orgsFound[orgID] = struct{}{}
 
+		if _, isDisabledOrg := moa.settings.UnifiedAlerting.DisabledOrgs[orgID]; isDisabledOrg {
```

Comment:
This should go above L152 to make sure we stop the alertmanger even if they had one - I don't think it matters given we require a restart to disable/enable but would be good to make this "semantically" correct.